skyportal/tests/frontend/test_shifts.py

```diff
@@ -346,7 +347,39 @@ def test_shift_summary(
     assert data['status'] == 'success'
 
     # wait for event to load
-    time.sleep(15)
+    n_times = 0
+    while n_times < 30:
```

Comment:
You know you can just do `for n_times in range(30)`?

Reply:
You're right ! Avoids having to declare and increment the variable. I'll do that then. When all the tests are green and the for loop is implemented, can I merge it or should i ask for you or Michael (a reviewer) to make that decision ? Idk if its ok for the person that created the PR to merge or if it should always be those giving a review.

Reply:
Done